tests/fixtures/scenarios/editorconfig/atmos.yaml

```diff
@@ -0,0 +1,396 @@
+    # Can also be set using 'ATMOS_COMPONENTS_HELMFILE_CLUSTER_NAME_PATTERN' ENV var
+    cluster_name_pattern: "{namespace}-{tenant}-{environment}-{stage}-eks-cluster"
+
+stacks:
+  # Can also be set using 'ATMOS_STACKS_BASE_PATH' ENV var, or '--config-dir' and '--stacks-dir' command-line arguments
+  # Supports both absolute and relative paths
+  base_path: "stacks"
+  # Can also be set using 'ATMOS_STACKS_INCLUDED_PATHS' ENV var (comma-separated values string)
+  included_paths:
+    - "orgs/**/*"
+  # Can also be set using 'ATMOS_STACKS_EXCLUDED_PATHS' ENV var (comma-separated values string)
+  excluded_paths:
+    - "**/_defaults.yaml"
+  # To define Atmos stack naming convention, use either `name_pattern` or `name_template`.
+  # `name_template` has higher priority (if `name_template` is specified, `name_pattern` will be ignored).
+  # `name_pattern` can also be set using 'ATMOS_STACKS_NAME_PATTERN' ENV var
+  # name_pattern: "{tenant}-{environment}-{stage}"
+  # `name_template` is a Golang template.
+  # For the template tokens, and you can use any Atmos sections and attributes that the Atmos command
+  # `atmos describe component <component> -s <stack>` generates (refer to https://atmos.tools/cli/commands/describe/component).
+  # `name_template` can also be set using 'ATMOS_STACKS_NAME_TEMPLATE' ENV var
+  name_template: "{{.vars.tenant}}-{{.vars.environment}}-{{.vars.stage}}"
+
+workflows:
+  # Can also be set using 'ATMOS_WORKFLOWS_BASE_PATH' ENV var, or '--workflows-dir' command-line argument
+  # Supports both absolute and relative paths
+  base_path: "stacks/workflows"
+
+logs:
+  # Can also be set using 'ATMOS_LOGS_FILE' ENV var, or '--logs-file' command-line argument
+  # File or standard file descriptor to write logs to
+  # Logs can be written to any file or any standard file descriptor, including `/dev/stdout`, `/dev/stderr` and `/dev/null`
+  file: "/dev/stderr"
+  # Supported log levels: Trace, Debug, Info, Warning, Off
+  # Can also be set using 'ATMOS_LOGS_LEVEL' ENV var, or '--logs-level' command-line argument
+  level: Info
+
+# Custom CLI commands
+commands:
+  # No arguments or flags are required
+  - name: "test"
+    description: "Run all tests"
+    steps:
+      - atmos vendor pull
+
+  # test for infinite loop
+  - name: loop
+    description: This command tests circuit breaker for infinite loop
+    steps:
+      - "echo Hello world!"
+      - atmos loop
+
+  - name: tf
+    description: Execute 'terraform' commands
+    # subcommands
+    commands:
+      - name: plan
+        description: This command plans terraform components
+        arguments:
+          - name: component
+            description: Name of the component
+        flags:
+          - name: stack
+            shorthand: s
+            description: Name of the stack
+            required: true
+        env:
+          - key: ENV_VAR_1
+            value: ENV_VAR_1_value
+          - key: ENV_VAR_2
```

Comment:
Most of the configuration in this file, has nothing to do with testing editor config. Can we reduce it to something minimal for testing editor config?

Reply:
Done